### DISCUSSION TITLE
problem signing in

### DISCUSSION THREAD

**Discussion by Parkmasterpro** [PrismLauncher/PrismLauncher#1360] [Q&A]
my freind cant sign in to prism using his microsoft acount on a steam deck. is there any way to fix this?

**Comment by pog5** [comment]
What error do they get? Have they tried using a phone/desktop to sign in using the code provided?

**Comment by Parkmasterpro** [comment]
I fixed it by changing his password

**Reply by TRTGGE** [reply]
Was it just your friend forgot his password or some other error? sorry to be a pain am just having a few problems myself so seeing if we found the same problem! :)
